torchbenchmark/models/soft_actor_critic/requirements.txt

```diff
@@ -0,0 +1 @@
+gym
```

Comment:
Is this a new torchbench requirement too?

Reply:
Yes, it will be a new torchbench requirement because `install.py` will recursively install the dependencies of all models.